releasenotes/notes/rename-user_parameters-to-training_parameters-7d8cf1e056b967a7.yaml

```diff
@@ -0,0 +1,16 @@
+---
+issues:
+  - |
+    Rename ``user_parameters`` in class ``QuantumKernel`` to ``training_parameters``.
+    This means rename the keyword-argument in the constructor, the properties
+    ``user_parameters``, ``user_param_binds``, the methods ``assign_user_parameters``
+    ``bind_user_parameters``, ``get_unbound_user_parameters``, to
+    ``training_parameters``, ``trainin_param_binds``, ``assign_training_parameters``
+    ``bind_training_parameters``, ``get_unbound_training_parameters``.
+
+deprecations:
+  - |
+    The existing keyword-argument ``user_parameters`` and the properties
+    ``user_parameters``, ``user_param_binds``, the methods ``assign_user_parameters``
+    ``bind_user_parameters``, ``get_unbound_user_parameters`` are all deprecated
+    and will be removed after three months.
```

Comment:
We should just end this after `are all deprecated` I think - the policy is no sooner than three months and it needs to be in there at least one more version so in practice it cannot be removed any sooner than 0.7.0 or 3 months, whichever comes first,

Reply:
done.